libpimeval/src/pimPerfEnergyModels.h

```diff
@@ -13,14 +13,7 @@
 #include "pimResMgr.h"
 
 
-//! @class  pimParamsPerf
-//! @brief  PIM performance parameters base class
-class pimParamsPerf
-{
-public:
-  pimParamsPerf(pimParamsDram* paramsDram);
-  ~pimParamsPerf() {}
-
+namespace pimNS {
```

Comment:
What does NS stand for? Also, could you please add some documentation on why this namespace was introduced and what to add under this and what not to add?

Reply:
I assume NS = Name Space? But as I'm reading the code, it seems to evoke the idea of nanoseconds (due to its common use). To avoid confusion, could we consider using something more descriptive like peval or pim?

Reply:
Renamed to pimeval for clarity